lib/send_sms.rb

```diff
@@ -0,0 +1,26 @@
+require 'twilio-ruby'
+
+class Sms
+  def initialize
+  end
+  def time
```

Comment:
Really like the method you have made to get the time that the takeaway is going to be delivered